tests/main/prepare-image-classic/task.yaml

```diff
@@ -0,0 +1,72 @@
+summary: Check that prepare-image can prepare UC20+ classic models
+
+# running this on one machine is enough
+systems: [ubuntu-22.04-64]
+
+environment:
+    ROOT: /home/test/tmp/
+    STORE_ADDR: localhost:11028
+    STORE_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    #shellcheck source=tests/lib/store.sh
+    . "$TESTSLIB"/store.sh
+    setup_fake_store "$STORE_DIR"
+
+    mkdir "$ROOT"
+    chown -R test:test "$ROOT"
+
+restore: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
```

Comment:
Is this check really neccessary at all steps for the test? I thought it was only neccessary for either prepare or execute.

Reply:
That is probably true, I just cargo culted this :-/

Reply:
I did a quick grep over the repo and we use this a lot (36 time unless I miscounted). Happy to remove again but maybe we should do a systematic sweep over the reo instead in a separate PR?

Reply:
I guess it's more important for the `execute` step only, but having it on the `prepare` and `restore` as well helps speeding up the test a bit, since the fake store doesn't need to be setup.